src/connectors/greathorn_logs.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import os
+import json
+from datetime import datetime
+from datetime import timedelta
+import time
+
+PAGE_SIZE = 200
+
+CONNECTION_OPTIONS = [
+    {
+        'name': 'api_key',
+        'title': "API Token",
+        'prompt': "This available in your GreatHorn",
+        'type': 'str',
+        'secret': True,
+        'required': True,
+    },
+]
+
+LANDING_EVENTS_TABLE_COLUMNS = [('raw', 'VARIANT', 'RECORDED_AT', 'TIMESTAMP_LTZ')]
+
+def connect(connection_name, options):
+    table_name = f'greathorn_{connection_name}'
+    landing_events_table = f'data.{table_name}_events_connection'
+
+    db.create_table(
+        name=landing_events_table,
+        cols=LANDING_EVENTS_TABLE_COLUMNS,
+        comment=yaml_dump(module='duo', **options),
```

Comment:
module will refer to the file that does the ingest -- on `greathorn_logs` for this one